DCRack/dc_rack_server.py

```diff
@@ -19,11 +19,9 @@
 name = DC Rack Server
 version = 2.2
```

Comment:
@jwenner this PR was sort of re-issued elsewhere. I think we may actually want to close this one.